### PR TITLE
LRAC-9627 watch color dropdown click for sending to Analytics

### PR DESCRIPTION
**Pull request by marcosapmf**
Related issue: https://issues.liferay.com/browse/LRAC-9627

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @marcosapmf, can you please check if these failures are related to your changes? Thanks

**Comment by marcosapmf** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ColorPicker/ColorPicker.es.js [hunk 1]
```diff
@@ -86,23 +86,76 @@ const ColorPicker = ({
...
-		name={name}
-		readOnly={readOnly}
-		spritemap={spritemap}
-		{...otherProps}
-	>
-		<ClayColorPickerWithState
-			inputValue={value ? value : predefinedValue}
+}) => {
+	let _event;
+
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ColorPicker/ColorPicker.es.js:91]: @marcosapmf, can we rename it to `colorDropDownClickEvent` instead?

In addition, can we remove the blank line on line 91?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ColorPicker/ColorPicker.es.js [hunk 2]
```diff
@@ -86,23 +86,76 @@ const ColorPicker = ({
...
+				}
+
+				previousShow = show;
+			}
+		}
+	});
+
+	// watch dropdown click for sending to Analytics
+
+	const observeColorDropDown = (event) => {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ColorPicker/ColorPicker.es.js:122]: @marcosapmf, can we rename it to `handleColorDropDownClicked` instead?

The goal behind this renaming is keep a consistency with the existing code. :wink:

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ColorPicker/ColorPicker.es.js (hunk 1 above)
Comment (marcosapmf) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ColorPicker/ColorPicker.es.js]: Thanks for reviewing, @natocesarrego. `colorDropDownClickEvent` looks much better :smile:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ColorPicker/ColorPicker.es.js (hunk 2 above)
Comment (marcosapmf) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ColorPicker/ColorPicker.es.js]: @natocesarrego, totally agreed :smile:

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @marcosapmf, can you please check if these failures are related to your changes? Thanks

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**